jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/resources/org/jbpm/workbench/pr/client/resources/i18n/Constants.properties

```diff
@@ -175,3 +175,4 @@ FromActivationTime=From activation time
 ShowInstanceBadges=Show instance badges
 TimerDelayHelp=In seconds
 DeploymentId=Deployment Id
+Default_Process_Definition_Name=Evaluation
```

Comment:
@bxf12315 too tied to our examples, could be something more generic like 'Name' ?

Reply:
ok